model/project.rb

```diff
@@ -28,10 +28,12 @@ class Project < Sequel::Model
   one_to_many :quotas, class: :ProjectQuota, key: :project_id
   one_to_many :invitations, class: :ProjectInvitation, key: :project_id
   one_to_many :api_keys, key: :owner_id, class: :ApiKey, conditions: {owner_table: "project"}
+  one_to_many :aws_location_credentials, key: :project_id, class: :AwsLocationCredential
```

Comment:
```suggestion
  one_to_many :aws_location_credentials
```

I think the values provided are what Sequel would chose as defaults.